kubernetes/README.md

```diff
@@ -76,6 +76,37 @@ These defaults can be modified using:
 --set cloudserver.autoscaling.config.targetCPUUtilizationPercentage=...
 ```
 
+Pod Disruption Budgets
+----------------------
+
+Zenko relies on several stateful services require a quorum to function properly
+as highly available services to provide resilency to many types of outage
+scenarios. Pod disruption budgets allow for configuration of how many pods of a
+given service can safely fail and continue to operate normally before Kubernetes
+disables access to the service.
+
+The following services have settings for configuring disruption budgets:
```

Comment:
I don't know if "services" is the right word to use when talking in a Kubernetes context because it might lead to confusion. Perhaps "application" is a better choice.

> Note: This applies to most uses of the word `service` in this PR.

Reply:
Yeah it's a tricky to navigate the "right" words to use. I'll try changing some of the wording.